src/type/preload.ts

```diff
@@ -606,6 +606,7 @@ export const configSchema = z
       .default({}),
     registeredEngineDirs: z.string().array().default([]),
     recentlyUsedProjects: z.string().array().default([]),
+    skipUpdateVersion: z.string().default(""),
```

Comment:
空欄に意味を持たせるのは危険かなと！
あと`rootMiscSettingSchema`側に移せばVuex経由が楽なのでそれもコメントさせてください。
```suggestion
    skipUpdateVersion: z.string().default(""), // FIXME: undefinedにする・rootMiscSettingSchemaに移す
```